Source/WebCore/platform/graphics/displaylists/DisplayListRecorderImpl.h

```diff
@@ -41,55 +41,69 @@ class RecorderImpl : public Recorder {
 
     bool isEmpty() const { return m_displayList.isEmpty(); }
 
+    void save(GraphicsContextState::Purpose) final;
```

Comment:
Why these function became public?

Reply:
The eventual intention is that something, at least a test, is able to call the functions directly.
Putting them as private is violation of the intention of the API:

```
Ref<DisplayListRecorderImpl> m_context;
GraphicsContextGL* a = m_context.get();
a->drawPath(..); // works
m_context->drawPath(..); // should work too! It's the same function. This is a better function to call in cases it is possible. 
```